### PR TITLE
Access DOM Element via ElementRef

### PR DESCRIPTION
**Pull request by alQlagin**

### CODE REVIEW

**Comment by alQlagin** [comment]
closes https://github.com/30-seconds/30-seconds-of-angular/pull/117

File: snippets/access-dom-element.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+links:
+- https://stackblitz.com/edit/angular-tooltipjs-directive?file=src%2Fapp%2Ftooltip.directive.ts
+- https://angular.io/api/core/ElementRef
+- https://angular.io/guide/security
+tags:
+- components
+- dom
+---
+# Content
+In rare cases when you need to access DOM element directly, you can get it by requiring `ElementRef` in your construtor.
```
Comment (fetis) [inline snippets/access-dom-element.md:14]: to make it more clear
```suggestion
In rare cases when you need to access DOM element of a component directly, you can get it by requiring `ElementRef` in your construtor. 
```

File: snippets/access-dom-element.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+```typescript
+@Directive({selector: '[tooltip]'})
+export class TooltipDirective implements OnInit, OnDestroy {
+  private readonly tooltip: Tooltip;
+  constructor(
+    private elementRef: ElementRef<HTMLElement>,
+  ) { }
+}
+```
+> Note: Try avoiding direct DOM manipulation except the cases when it's the only option, such as interaction with 3rd party libraries.
```
Comment (fetis) [inline snippets/access-dom-element.md]: do we need a quote?

```suggestion
Note: Try avoiding direct DOM manipulation except the cases when it's the only option, such as interaction with 3rd party libraries.
```

File: snippets/access-dom-element.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+# file:app.component.ts
+```typescript
+import { Component } from '@angular/core';
+
```
Comment (fetis) [inline snippets/access-dom-element.md:29]:
```suggestion
```

File: snippets/access-dom-element.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+import { Attribute, Directive, ElementRef, OnInit, OnDestroy, HostListener } from '@angular/core';
+import Tooltip from 'https://unpkg.com/popper.js';
+
+@Directive({
+  selector: '[tooltip]'
+})
+export class TooltipDirective implements OnInit, OnDestroy {
+  
+  private tooltip: Tooltip;
+  constructor(
```
Comment (fetis) [inline snippets/access-dom-element.md:39]:
```suggestion

  constructor(
```

**Review by fetis — APPROVED** [review]

**Pull request merged**